rosdep/python.yaml

```diff
@@ -1843,6 +1843,19 @@ python-graphviz-pip:
 python-gridfs:
   debian: [python-gridfs]
   fedora: [python-pymongo-gridfs]
+python-grip:
+  debian:
+    pip:
+      packages: [grip]
```

Comment:
It looks like this is available on (some) Debian releases as a system package: https://packages.debian.org/search?keywords=grip&searchon=names&suite=all&section=all .  Please fix it to use the system package where possible.

Reply:
Thank you! I updated.